a.html

```diff
@@ -0,0 +1,102 @@
+      <div class="search">
+        <form action="">
+          <input type="text" placeholder="search"/>
+          <button>
+              <i class="material-icons">search</i>
+          </button>
+      </form>
+      <i class="material-icons">mic</i>
+      </div>
+      <div class="header_icons">
+        
+        <i class="material-icons">videocam</i>
+        <i class="material-icons">apps</i>
+        <i class="material-icons">notifications</i>
+        <i class="material-icons">account_circle</i>
+    </div>
+
+    </header>
+    <main>
+     
+        <div class="content">
+          <div class="thumbnails">
+            <div class="col">
+             <div class="small-img-row">
+               <div class="small-img">
+                 <img src="yellow ladybird.jpg">
+               </div>
+               <p>yellow with black dottes<br>can also fly</p>
+             </div>
+           </div>
+           <div class="thumbnails">
+             <div class="col">
+              <div class="small-img-row">
+                <div class="small-img">
+                  <img src="red ladybird.jpeg">
+                </div>
+                <p>yellow with black dottes<br>can also fly</p>
+              </div>
+            </div>
+            <div class="thumbnails">
+             <div class="col">
+              <div class="small-img-row">
+                <div class="small-img">
+                  <img src="whitedotted.jpeng.jpg">
+                </div>
+                <p>yellow with black dottes<br>can also fly</p>
+              </div>
+            </div>
+          <iframe width="800" height="415" src="https://www.youtube.com/embed/ws_D5nXOAJg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        
+          <h2>Insect life circle</h2>
+          <h4>A full life circle from eggs to fully developed lady bird</4>
+          <a href="">ladybird life circle</a><span>1.2m Views.in 2 years</span>
+          <div class="footer-icons">
+            <i class="material-icons">thumb_up</i><span>Likes</span>
+        <i class="material-icons">thumb_down_alt</i><span>Dislikes</span>
+        <i class="material-icons">share</i><span>share</span>
+        <i class="material-icons">save_alt</i><span>save</span>
+        <div class="subscribe">
+          <form action="">
+            <input type="text" placeholder="2000 subcribers"/>
+            <button>
+                <i class="SUBSCRIBE">SUBSCRIBE</i>
+            </button>
+        </form>
+
+      </div>
+      <!-- side bar -->
+      
+          <div class="comments">
+            <h3>add your comment....</h3>
+            <form action="">
+              <textarea name="comments" placeholder="type your comment"></textarea>
+            
+                  <button class="Submit">Submit</button>
+                  </form>
+          </div>
+        </div>
+      </div>
```

Comment:
the container of this section should be a <aside>tag for better semantic HTML. Kindly replace div with <aside> tag.